### ISSUE TITLE
Document Crowdmap Classic

### ISSUE THREAD

**Issue by rjmackay** [ushahidi/platform#1423]
Drop as much as you know in a google doc.

**Comment by jshorland** [comment]
@rjmackay can you drop link to the docs in here?

**Comment by rjmackay** [comment]
https://drive.google.com/open?id=0B0A6Dk9aXj5oR09SOVk0SjdNeEE

**Comment by rjmackay** [comment]
@tuxpiper did you get a chance to review the docs?

**Comment by tuxpiper** [comment]
@rjmackay just going through them, hopefully I will be done in the next couple days

**Comment by tuxpiper** [comment]
Ok @rjmackay that was quicker than I thought, I've added comments. It looks like most immediately we've got:

1. A few items to clarify
2. Some basic cleaning up to do (i.e. ensuring server access, repossessing the crowdmapid repo from Evan)

A note about costs. At a first glance, it doesn't look like we can save a lot of money (say 50%) by powering down replicated servers. My wild guess is that so much money goes into hosting because the individual servers may be overprovisioned. I guess we could look at detailed costs as part of the clean up effort.
